src/js/DropContainer.jsx

```diff
@@ -0,0 +1,281 @@
+      // But Chrome implements some drag store, which is accesible via dataTransfer.items
+      const dataTransferItems = this.extractData(e);
+
+      // Now we need to convert the DataTransferList to Array
+      const allFilesAccepted = this.fileAccepted(Array.prototype.slice.call(dataTransferItems));
+
+      this.setState({
+         isDragActive: allFilesAccepted,
+         borderStyle: 'dragBorder',
+         visibility: 'hide'
+         // isDragReject: !allFilesAccepted
+      });
+
+      if (allFilesAccepted) {
+         this.successBorder()
+      } else {
+         this.warningBorder();
+      }
+  }
+
+   /**
+      * Handle the drag over event
+      * @param  {Event]} event
+   */
+   onDragOver (e) {
+      e.preventDefault();
+      e.stopPropagation();
+      e.dataTransfer.dropEffect = 'copy';
+
+      return false;
+   }
+
+   /**
+      * Handle the drag leave event
+   */
+   onDragLeave (e) {
+      e.preventDefault();
+
+      // Only deactivate once the dropzone and all children was left.
+      if (--this.enterCounter > 0) {
+         return;
+      }
+
+      this.setState({
+         isDragActive: false,
+         isDragReject: false,
+         borderStyle: '',
+         visibility: 'show'
+      });
+
+      if (this.props.onDragLeave) {
+         this.props.onDragLeave.call(this, e);
+      }
+
+      if(this.state.imageFile){
+         // console.log('onDragLeave');
+         return;
+      }
+  }
+
+   /**
+      * Handle the drop event
+      * @param  {Event]} event
+   */
+   onDrop (e) {
+      e.preventDefault();
+      this.setState({
+         isDragActive: false,
+         borderStyle: '',
+         visibility: 'show'
+      });
+      this.handleChange.call(this, e);
+   }
+
+   /**
+      * Handle the click event on drop container
+   */
+   // onClick () {
+   //    console.log('on click called');
+   //    this.refs.fileInput.myTextInput.click();
+   // }
+
+   /**
+      * Handle the image state
+      * If image is available, will create a URL obj
+      * And modify the state value
+      * @param  {Event]} event
+   */
+   handleChange (event) {
+      let e = event.nativeEvent, imageFile;
+
+      const dataTransferItems = this.extractData(event);
+      const allFilesAccepted = this.fileAccepted(Array.prototype.slice.call(dataTransferItems));
+
+      // Check to make sure the dropped file
+      // is in image format - image/*
+      if(allFilesAccepted){
+         // Setting the state image source
+         // With drag & drop event
+         if(e.dataTransfer) {
+            let evt = e.dataTransfer.files[0];
+
+            if(evt !== undefined) {
+               imageFile = URL.createObjectURL(evt);
+               // console.log('**DRAGGED** SELECTED FILE', evt);
+               this.setState({
+                  imageFile: imageFile
+               });
+            } else {
+               console.warn('Warning: no image selected');
```

Comment:
This should probably be communicated in a way the user can see. Console lines are striped out typically in production as well.